h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -128,11 +129,18 @@ def extend_parms(parms):
 overrides = dict(
     base_models=dict(
         setter="""
-if is_type(base_models, {ptype}):
-    {pname} = [b.model_id for b in {pname}]
+def _get_id(something):
+    if hasattr(something, "key"):
```

Comment:
strictly speaking, I'd prefer `if isinstance(something, Keyed)`, I know it's not following duck-typing rules, but `key` is a pretty common property, and only classes extending `Keyed` have it with the correct semantic.